src/MvpRoutes.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Switch, Route } from 'react-router';
+
+import Index from './pages/Index';
+import ApiAccessTokens from './pages/ApiAccessTokens';
+import Tokens from './pages/Tokens';
+import ProfilePage from './pages/ProfilePage';
+import Header from './components/Header';
+import config from './config';
+
+function MvpRoutes(): React.ReactElement {
```

Comment:
Are two route components really needed (`MvpRoutes` and `PlainSuomiFiRoutes`)? They are pretty similar. The current configuration's `path` could be used in the matches, as it currently also is. The configuration just happens to be hard coded in these two separate components now, but couldn't it also be fetched dynamically?

Reply:
I thought it made App.tsx much clearer and easier to add/modify routes. 
MvpRoutes and PlainSuomiFiRoute are now removed.